### DISCUSSION TITLE
Petnet vs Orchestration Layer

### DISCUSSION THREAD

**Discussion by Cosmodude** [NillionNetwork/discussions#161] [Q&A]
I was researching about Nillion Network a lot recently. Read the documentation and articles. 

One of them mentioned the so-called "Orchestration Layer" and had an overview of its architecture. https://nillion.com/news/564/

However, I couldn't find anything about it in the docs. Was it deprecated/refactored?
Otherwise, what's its relation to PetNet?

It looks like the "Coordination Layer" was renamed as the NilChain, so I wonder if Petnet is another name for the "Orchestration Layer"

Thank you!

**Comment by crypblizz8** [comment]
Hi @Cosmodude, thanks for looking into Nillion.

This is the most up to date information on the architecture with our updated docs
https://docs.nillion.com/learn/architecture

We refer it to PetNet and NilChain

**Comment by Cosmodude** [comment]
After a discussion with one of the Nillion team members: 

> The Petnet is just what we call the side of the network that has the clusters that perform PET operations
> 
> Orchestration layer is really more of a concept, but it’s the idea of plugging in different PETs together to make the products/modules work

So, the Orchestration Layer kind of emphasizes Petnet's role in uniting PETs.

+

that's what NIllion AI gave me 
![photo_2025-01-25 21 04 04](https://github.com/user-attachments/assets/789068b2-d476-435b-9a72-c1defe229a60)
